src/test/bech32_tests.cpp

```diff
@@ -2,7 +2,7 @@
 // Distributed under the MIT software license, see the accompanying
 // file COPYING or http://www.opensource.org/licenses/mit-license.php.
 
-#include <bech32.h>
+#include <bech32.cpp>
```

Comment:
Yuck (including a .cpp file).

Reply:
*chants* export export export

Reply:
linter doesn't like it either :D

Reply:
Yeah was to test the internal methods. Changed that meanwhile.